### PR TITLE
Feat/review storage

### PR DESCRIPTION
**Pull request by mailo001**

### CODE REVIEW

File: server/pkg/bookly/review.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	Content    string
+	Rating     int32 //[1-5]
+	ReviewDate time.Time
+}
+
+// ReviewStorage is responsible for persisting review
+type ReviewStorage interface {
+	CreateReview(ctx context.Context, review Review) (int64, error)
+	DeleteReview(ctx context.Context, reviewID int64) error
+	DeleteReviewByOwner(ctx context.Context, userID int64, offerID int64) error
```
Comment (Gloomwidow) [inline server/pkg/bookly/review.go:23]: Can't we just check if review belongs to user and delete it by ID? Function for checking review ownership could be useful in services.

File: server/pkg/bookly/review.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	GetReview(ctx context.Context, reviewID int64) (Review, error)
+	GetReviewByOwner(ctx context.Context, userID int64, offerID int64) (Review, error)
```
Comment (Gloomwidow) [inline server/pkg/bookly/review.go:25]: Same as above

File: server/pkg/bookly/review.go [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	GetReviewsOfOffer(ctx context.Context, offerID int64) ([]*Review, error)
+}
+
+// ReviewService is a service which is responsible for actions related to review
+type ReviewService interface{}
+
+// ErrReviewNotFound indicates that review doesn't exist in database
+var ErrReviewNotFound = errors.New("review not fount")
```
Comment (Gloomwidow) [inline server/pkg/bookly/review.go:33]: This should be placed in `error.go`

File: server/pkg/bookly/review.go (hunk 1 above)
Comment (areknoster) [reply server/pkg/bookly/review.go:23]: Great idea

**Pull request merged**